src/extension/index.ts

```diff
@@ -63,8 +64,11 @@ export async function activate(context: ExtensionContext) {
                     const pair = param.split('=');
                     if (pair[0] === 'url') {
                         // Decode the alert API URL and pass it to the load function.
-                        await loadAlertSarif(new URL(decodeURIComponent(pair[1])));
-                        break;
+                        const url = decodeURIComponent(pair[1]);
+                        if (url.startsWith('https://advsec.dev.azure.com/')) {
```

Comment:
If the url is incorrect, should there be an error thrown?

Reply:
Sure, any suggestions on the message? It will be shown on the banner in the lower right corner. I'm not sure how much detail to provide, e.g. what the host must be, etc.

Reply:
How about something like:

> `Invalid URL '${url}'. Expecting a URL that starts with 'https://advsec.dev.azure.com/'.`

I am not sure how much control a user has over the URL here. If the URL is invalid, does this mean that there is an invalid alert?

Reply:
Nope, just means the one they clicked on passed a URL that doesn't point to advsec.

Reply:
Ok. Makes sense. I still think a message would be useful so the user knows why the link wasn't opened.